doc/nrf/config_and_build/bootloaders/bootloader_external_flash.rst

```diff
@@ -13,14 +13,19 @@ When using MCUboot, you can store the storage partition for the secondary slot i
 * Single-byte read and write.
 * Writing data from the internal flash memory to the external flash memory.
 
+MCUboot typically increases slot sizes when external flash is enabled.
```

Comment:
Users have to do this as a consequence of enabling external flash.
I suggest we desrcibe CONFIG_BOOT_MAX_IMG_SECTORS after we have told users how to enable external flash. Such as at the end of this section.